src/test/java/uk/gov/pay/ledger/event/model/TransactionEntityFactoryTest.java

```diff
@@ -103,4 +108,34 @@ public void fromShouldConvertEventDigestToTransactionForChildResource() {
         assertThat(transactionEntity.getParentExternalId(), is(parentResourceExternalId));
         assertThat(transactionEntity.getState(), is("submitted"));
     }
+
+    @Test
+    public void fromShouldRejectEventDigestWithoutStateTransitionEvents() throws Exception {
```

Comment:
are we calling them `salient` or `statetransition` events

Reply:
have updated everything back to `Team Salient`